packages/adblocker/src/config.ts

```diff
@@ -26,6 +26,7 @@ export default class Config {
       loadExtendedSelectors: buffer.getBool(),
       loadGenericCosmeticsFilters: buffer.getBool(),
       loadNetworkFilters: buffer.getBool(),
+      loadPreprocessors: buffer.getBool(),
```

Comment:
Based on the name of this option alone I think it is a bit ambiguous what happens when this is `false`:
1. Does it mean the preprocessor blocks (and their content) will be ignored; meaning that the filters inside of the blocks will not be loaded in the engine.
2. Does it mean instead that the preprocessor directives are ignored while parsing and that all filters are loaded into the engine regardless?

Reply:
When it is set to `false`, I intended it to parse and register preprocessors but doesn't evaluate the conditions at all. So the set of filters should be always empty. However, it should be clearer. I'll try to make this less opaque.